### ISSUE TITLE
Add JS trigger to `edd_load_gateway()` for third-party hooks

### ISSUE THREAD

**Issue by jplhomer** [easydigitaldownloads/easy-digital-downloads#4146]
It would be nice to have a hook to use for third-party plugins/themes when loading in a gateway over AJAX. This would enable input validation for a given payment gateway form.

E.g.:

```js
// Format CC input using Stripe's jquery.payment library
$('body').on('edd_gateway_loaded', function( event, payment_mode ) {
    if ( 'stripe' === payment_mode) {
        $('input#card_number').payment('formatCardNumber');
    }
});
```

I'll work up a pull request for this now!

**Comment by pippinsplugins** [comment]
@jplhomer Love the idea :)

**Comment by cklosowski** [comment]
Since we have a PR, going to milestone for 2.6, we can punt if necessary, but I rather like this.

**Comment by pippinsplugins** [comment]
I like this.

**Comment by pippinsplugins** [comment]
Merged into release/2.6
